### ISSUE TITLE
expanding to RGCN 

### ISSUE THREAD

**Issue by jzwart** [jzwart/da_pgdl_stream_temp#18]
I have written some code to incorporate RGCN, but there have been some additions to LSTM that needs to be incorporated when predicting with RGCN (e.g. mc dropout)

**Comment by jzwart** [comment]
RGCN training takes much longer - ~20x's longer / epoch.

**Comment by jzwart** [comment]
Having some issues with getting RGCN to converge on accurate predictions in the training phase - training RMSE hanging out around 6-7 C for temperature. Currently using 20 hidden units, 30 epochs pre-train, 250 epochs fine tune, 4 connected stream segments. Most likely a bug somewhere in the code and need to revisit later
